numpy/__init__.pxd

```diff
@@ -695,6 +696,22 @@ cdef extern from "numpy/arrayobject.h":
     int PyArray_SetBaseObject(ndarray, base)  # NOTE: steals a reference to base! Use "set_array_base()" instead.
 
 
+cdef extern from "numpy/ndarrayobject.h":
+    PyTypeObject PyTimedeltaArrType_Type
+    PyTypeObject PyDatetimeArrType_Type
+
+cdef extern from "numpy/ndarraytypes.h":
+    ctypedef struct PyArray_DatetimeMetaData:
+        NPY_DATETIMEUNIT base
```

Comment:
Something is missing: this line fails with `/home/circleci/repo/numpy/__init__.pxd:705:8: 'NPY_DATETIMEUNIT' is not a type identifier`

Reply:
Looks like I had the declarations in the wrong order, just rearranged them and pushed, so we'll find out in a bit.